README.md

```diff
@@ -42,6 +42,25 @@ hooks('ts').push(function({content}){
 /* transpile your script code here */
 });
 ```
+
+For custom blocks in `.vue` files, add other require hooks for the name of the blocks,
```

Comment:
Can you give this section its own header? I think it's a big-enough (and optional) feature that it deserves its own section